index.js

```diff
@@ -6,14 +6,13 @@ var AWS = require("aws-sdk"),
 	fs = require("fs"),
 	request = require("request"),
 	path = require("path"),
+	sharp = module.parent.require("sharp"),
```

Comment:
I may be wrong but this looks like it would try and require sharp from nodebb not the s3-uploads package?